x/hold/spec/03_messages.md

```diff
@@ -0,0 +1,3 @@
+# Msg Service
```

Comment:
Can we just remove this spec?

Reply:
Yeah. I originally had it because I wanted it to be clear that there aren't any Tx endpoints as opposed to their documentation just missing. I poked around in the SDK, though, and they don't have any such mostly-empty pages.

Good call. Removed.